src/api/spec/models/staging_workflow_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
-RSpec.describe StagingWorkflow, type: :model do
-  let(:project) { create(:project_with_package) }
+RSpec.describe StagingWorkflow, type: :model, vcr: true do
```

Comment:
Why is this necessary?

Reply:
If you are talking about the VCR, is because we store the XML file in the backend when we create/update a staging_workflow

Reply:
I will check it